lib/heroku/kensa/check.rb

```diff
@@ -281,9 +282,11 @@ def call!
           :heroku_id => heroku_id,
           :plan => data[:plan] || 'test',
           :callback_url => callback,
+	  :log_input_url => "https://token:t.01234567-89ab-cdef-0123-456789abcdef@1.us.logplex.io/logs",
```

Comment:
I believe this field is not actually always present, but I think we should add a test case that runs for add-ons which have this enabled.